tests/codebuild/create_tests.sh

```diff
@@ -127,7 +128,7 @@ function verify_integration_tests
 # Replaces the names of the endpoint generated in the previous test into the hap spec file and runs the test. 
 # Parameter:
 #    $1: Target namespace
-#    $2: K8s Name of the hostingdeployment to apply autoscaling 
+#    $2: K8s Name and endpointName of the hostingdeployment to apply autoscaling 
 #    $3: Filename of the hap test to run
 #    $4: Filename of the custom metric hap test to run
 function run_hap_test()
```

Comment:
this function is much cleaner now thanks 

why not go one more step further add `create_hosting_deployments $number` function ?
```
create_hosting_deployments() {
  # This function creates specified number of hosting deployments from <...>.yaml
  # with name = (endpoint_name_prefix)_(number_of_endpoints)

  endpoint_name_prefix="$1"
  number_of_endpoints="$2"

  if [[ $number_of_endpoints -eq 0 ]]; then
    return
  fi

  echo code to create hostingdeployment "$deployment_name_prefix_$number_of_endpoints"
  create_hosting_deployments "$deployment_name_prefix" $((number_of_endpoints-1))
  echo code to verify hostingdeployment "$2"
}

echo Creating 3 endpoints
create_hosting_deployments endpoint_prefix 3
```
```
Creating 3 endpoints
code to create hostingdeployment 3
code to create hostingdeployment 2
code to create hostingdeployment 1
code to verify hostingdeployment 1
code to verify hostingdeployment 2
code to verify hostingdeployment 3
```